source/about/self-hosted-subscriptions.rst

```diff
@@ -99,13 +99,25 @@ To send this report, take a screenshot of **System Console > Site Statistics** a
 - We don't need your server address, so if it appears on your screenshot, you can redact it from the image.
 
 .. tip:: 
-   
-   Not sure where to take the screenshot? Please reach out to your account executive, Customer Success Manager, orders@mattermost.com, or support@mattermost.com for help.
-   
+
+   Not sure where to find the true-up review section? Please reach out to your account executive, Customer Success Manager, orders@mattermost.com, or support@mattermost.com for help.
```

Comment:
"True-up review section" might not be something they're immediately familiar with. Recommend using "Site Statistics" which is in the System Console and searchable